nix/sources.json

```diff
@@ -0,0 +1,102 @@
+{
+    "clash-compiler": {
+        "branch": "1.8",
+        "description": "Haskell to VHDL/Verilog/SystemVerilog compiler",
+        "homepage": "https://clash-lang.org/",
+        "owner": "clash-lang",
+        "repo": "clash-compiler",
+        "rev": "3f5dc67c0e526e43a4dd88eb3902e39ed512c166",
+        "sha256": "022rwif8xkaabw0j3arhyj0hswmh3vq2nx1bbz8rbkp05jm4psgg",
+        "type": "tarball",
+        "url": "https://github.com/clash-lang/clash-compiler/archive/3f5dc67c0e526e43a4dd88eb3902e39ed512c166.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz",
+        "version": "1.8.1"
```

Comment:
Do we want this repository to track Clash stable by default? I don't know, I'm asking :-).

Reply:
Copied from the clash starter. I don't spend any time thinking about it.